magento/adobe-stock-integration
# Dynamically create folder by current_tree_path when instance changed

### Description (*)

Dynamically create folder by current_tree_path when instance changed

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1490:"wysiwyg" folder is not created after the "catalog/category" has been created 
2. ...

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery** and verify that there are no _catalog/category_ and _wysiwyg_ folders. If the folders exist, delete them
2. Go to **Catalog - Categories**
3. Expand **Content**, click **Select from Media Gallery**
4. Verify that the _catalog/category_ folder is created and selected 
![cat_cat](https://user-images.githubusercontent.com/45624059/84896228-3fd4de00-b0ac-11ea-9765-08b4b2a53224.png)
5. Close (X) Media Gallery. DO NOT REFRESH the web page
6. Click **Insert/edit image** from **Description** editor
![ins_edit](https://user-images.githubusercontent.com/45624059/84896490-b7a30880-b0ac-11ea-9458-6b34aa01e9ed.png)
7. Click search source from _Insert/edit image_ window
![search_source](https://user-images.githubusercontent.com/45624059/84896622-ed47f180-b0ac-11ea-9fad-95aa33c53b10.png)

### Expected result (*)
_wysiwyg_ folder is created and selected
![wys_folder](https://user-images.githubusercontent.com/45624059/84896806-43b53000-b0ad-11ea-837d-b85ea73ca01f.png)
